### PR TITLE
docs: updating batch_importing_MARC.adoc

### PR DESCRIPTION
**Pull request by gmontimantis**
Some areas need updating but I did a decent chunk of it.  Will submit an LP ticket once this is merged.

### CODE REVIEW

File: docs/modules/cataloging/pages/batch_importing_MARC.adoc [hunk 1]
```diff
@@ -57,7 +57,7 @@ set for authority records or bibliographic records.
 
 . Click *Save*.
 +
-image::media/Batch_Importing_MARC_Records1.jpg[Batch_Importing_MARC_Records1]
+image::batch_importing_MARC/new_record_match.jpg
```
Comment (sandbergja) [inline docs/modules/cataloging/pages/batch_importing_MARC.adoc:60]: These image macros will need to end with a set of square brackets, per the [antora documentation](https://docs.antora.org/antora/2.2/asciidoc/insert-image/#block-image-macro-structure).  Better still would be to add some alt text into those brackets, for example:

    image::batch_importing_MARC/new_record_match.jpg["the new record match form, where a user has entered a match set type, name, and owning library for their new record matching definition set"]

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks for the PR, @gmontimantis.  The only issue seems to be that, when I build the docs, the images are not getting included into the final product.  I think this is because you are missing the square brackets at the end of the `image` macro.  Could you please add these?

Also, worth noting that you can add alt text for the images in those brackets.  The alt text we had before wasn't particularly helpful, so definitely some opportunity to improve upon it if you're willing to do so.

Also, no need to file a ticket in launchpad -- just handling this in github should be just fine. :-)

File: docs/modules/cataloging/pages/batch_importing_MARC.adoc (hunk 1 above)
Comment (gmontimantis) [reply docs/modules/cataloging/pages/batch_importing_MARC.adoc]: I tried doing a push to origin via GitHub desktop.  Let me know if you received those.  If not, I'll try again.

**Comment by sandbergja** [comment]
Works well for me.  Thanks for these long-needed updates, @gmontimantis!  Pushed to master, rel_3_7, and rel_3_8.

**Pull request closed without merging**